docs/contributing.rst

```diff
@@ -163,3 +163,15 @@ Preparing Pull Requests
      $ git rebase upstream/master  # Apply the changes that have been made to central repo,
      $ # since your last fetch, onto you master.
      $ git branch -d your-bugfix-feature-branch-name  # to delete the branch after PR is approved
+
+Release Instructions
+--------------------
+
+This is a documentation repo for people in the group on how to do the integrated deployment.
+
+NB RULE! Never commit to master.
+
+1. Change the version in the setup.py file. Must be format YYYY.<release number>
+2. Create a release with a tag that has the same format as the version above.
+3. The distribution will be built automatically and pushed to PyPi
+4. The DOI will also be updated on Zenodo.
```

Comment:
I don't think the automatic zenodo update is working atm, if it ever did. Issue #165 details current status. Last upload to z
enodo was in 2019 :/

Reply:
According to #165, Julius activated the function to update Zenodo DOIs. This PR aims at removing the unnecessary repository and moving the content into the ReadTheDocs instead.  
For the Zenodo DOI stuff, there is probably one way to test it: Creating a new release.

Reply:
cool, may as well give it a shot!